frontend/docs/components.md

```diff
@@ -0,0 +1,60 @@
+# Components
+
+## Design system functional components
+
+Functional components are components with no state which are styled according to the design system and used to compose higher level components.
+
+Those components live in `@/components/design-system/:category/ComponentName.vue` SFC files.
+
+A minimal component may look like this:
+
+```vue
+<!-- Script setup section, using TypeScript -->
+<script setup lang="ts">
+// Imports
+import { computed } from 'vue';
+
+// Props (interface) definition
+export interface Props {
```

Comment:
optional props by meli byt jenom tam kde je to opravdu optional. davat vsechny optional mi neprijde dobry ale na tom se asi neshodnem :)